compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmValueGen.java

```diff
@@ -230,20 +221,22 @@ void generateValueClasses(Map<String, byte[]> jarEntries, JvmConstantsGen jvmCon
                         , asyncDataCollector);
                 jarEntries.put(className + ".class", bytes);
             } else if (bType.tag == TypeTags.RECORD) {
+                JvmTypeGen jvmTypeGen = new JvmTypeGen(jvmConstantsGen, module.packageID, typeHashVisitor,
+                        jvmPackageGen.symbolTable);
```

Comment:
Can't we pass the existing `jvmTypeGen` from the upper level (`JvmPackageGen.generate` method) without creating new object here?